src/index.html

```diff
@@ -17,6 +20,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
It's a good practice to add `role="timer"` to the main stopwatch container for better accessibility. This helps screen readers and other assistive technologies understand the purpose of the element.